course_catalog/etl/enrollments.py

```diff
@@ -30,9 +30,10 @@ def update_enrollments_for_user(user):
         SELECT 
             enrollment_course_id,
             enrollment_created
-        FROM ol_warehouse.mitx_enrollments 
+        FROM {database}.mitx_enrollments 
         WHERE email = '{email}'
         """.format(
+        database=settings.ATHENA_WORK_GROUP,
```

Comment:
This should be `settings.ATHENA_MITX_DATABASE_NAME`. We might also want to make the table name configurable as it will likely evolve/change as we add more enrollments data sources.